requirements-dev.txt

```diff
@@ -52,4 +52,4 @@ sqlalchemy
 xarray
 xlrd
 xlsxwriter
-xlwt
\ No newline at end of file
+xlwt
```

Comment:
Not sure what's happening here.  Can you revert all changes to this file?  You can do just:

`git checkout master requirements-dev.txt`

Reply:
In the end I soft reset my branch to previous commit, removed requirements-dev.txt from it and force pushed. I believe it should be fine now.